README.md

```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/xenserver/xe-guest-utilities.svg?branch=master)](https://travis-ci.org/xenserver/xe-guest-utilities)
 
-go-guest-utilites
+Introduction
 ===================
 
 This is the golang guest utilities for XenServer
```

Comment:
This is the golang guest utilities for XenServer -> This is the golang guest utilities for Citrix Hypervisor

Reply:
Raise another PR to fix all these including other citrix related name